apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java

```diff
@@ -711,6 +724,14 @@ private void openStreamContext(StreamContext streamContext) throws IOException {
         StreamConfiguration requestedConfig = streamContext.tester.requestedConfiguration;
         StreamConfiguration actualConfig = streamContext.tester.actualConfiguration;
 
+        if (requestedConfig.getFormat() == StreamConfiguration.AUDIO_FORMAT_MP3 &&
+                (requestedConfig.getDirection() != StreamConfiguration.DIRECTION_OUTPUT ||
+                        requestedConfig.getChannelMask() != MP3_FILE_CONFIG.mChannelMask ||
+                        requestedConfig.getSampleRate() != MP3_FILE_CONFIG.mSampleRate)) {
+            showErrorToast("MP3 format uses builtin 44.1kHz mono mp3 file for playback");
```

Comment:
this should be stereo. Perhaps we can change the wording to "the test file for MP3 format uses 44.1kHz stereo mp3 file for playback"